adaptor-rm-webapp/src/main/java/com/sample/rm/RMToolManager.java

```diff
@@ -177,26 +186,78 @@ public static void contextDestroyServletListener(ServletContextEvent servletCont
     {
         List<Requirement> resources = null;
         
+        // Start of user code queryRequirements_storeInit
+        // End of user code
+        Store store = storePool.getStore();
+        try {
+            resources = new ArrayList<Requirement>(store.getResources(storePool.getDefaultNamedGraphUri(), Requirement.class, "", "", "", limit+1, page*limit));
```

Comment:
We actually need to add ORDER BY in https://github.com/eclipse/lyo.store/blob/master/src/store-core/src/main/java/org/eclipse/lyo/store/internals/SparqlStoreImpl.java#L482 for this to work reliably.

Reply:
Ok. But note that I am using constructSparqlWhere more and more, instead of modelFromQueryFlatPaged.
https://github.com/eclipse/lyo.store/blob/ddc3b43f8eb152df6b751ab1ab492998c9be1659/src/store-core/src/main/java/org/eclipse/lyo/store/internals/SparqlStoreImpl.java#L500

Reply:
I'm closing this issue here, because we are catching it under Store https://github.com/eclipse/lyo.store/issues/58